### ISSUE TITLE
Serialization of CorpusSchema gives wrong primary keys

### ISSUE THREAD

**Issue by shuttle1987** [persephone-tools/persephone-web-API#58]
Consider this script to create a corpus via Curl:

```sh

# This is a very quick and dirty to populate some initial data via calling the API.
# Note that for now ID's are hardcoded in later steps.
# TODO: process response data IDs
# Upload WAV files
echo "*** uploading WAV files ***"
curl -X POST --header 'Content-Type: multipart/form-data' --header 'Accept: application/json' --form audioFile=@crdo-NRU_F4_ACCOMP_PFV.1.wav 'http://127.0.0.1:8080/v0.1/audio'
curl -X POST --header 'Content-Type: multipart/form-data' --header 'Accept: application/json' --form audioFile=@crdo-NRU_F4_ACCOMP_PFV.3.wav 'http://127.0.0.1:8080/v0.1/audio'
curl -X POST --header 'Content-Type: multipart/form-data' --header 'Accept: application/json' --form audioFile=@crdo-NRU_F4_ACCOMP_PFV.7.wav 'http://127.0.0.1:8080/v0.1/audio'
# Upload transcriptions
echo "*** uploading transcription files ***"
curl -X POST --header 'Content-Type: multipart/form-data' --header 'Accept: application/json' --form transcriptionFile=@crdo-NRU_F4_ACCOMP_PFV.1.phonemes 'http://127.0.0.1:8080/v0.1/transcription'
curl -X POST --header 'Content-Type: multipart/form-data' --header 'Accept: application/json' --form transcriptionFile=@crdo-NRU_F4_ACCOMP_PFV.3.phonemes 'http://127.0.0.1:8080/v0.1/transcription'
curl -X POST --header 'Content-Type: multipart/form-data' --header 'Accept: application/json' --form transcriptionFile=@crdo-NRU_F4_ACCOMP_PFV.7.phonemes 'http://127.0.0.1:8080/v0.1/transcription'
# Create Utterances
echo "*** specifying utterances ***"
curl -X POST --header 'Content-Type: application/json' --header 'Accept: application/json' -d '{
   "audioId": 1,
   "transcriptionId": 1
 }' 'http://127.0.0.1:8080/v0.1/utterance'
curl -X POST --header 'Content-Type: application/json' --header 'Accept: application/json' -d '{
   "audioId": 2,
   "transcriptionId": 2
 }' 'http://127.0.0.1:8080/v0.1/utterance'
curl -X POST --header 'Content-Type: application/json' --header 'Accept: application/json' -d '{
   "audioId": 3, 
   "transcriptionId": 3 
 }' 'http://127.0.0.1:8080/v0.1/utterance'

# Create corpus
echo "*** creating corpus ***"
curl -X POST --header 'Content-Type: application/json' --header 'Accept: application/problem+json' -d '{
   "name": "Test Corpus", 
   "label_type": "phonemes",
   "feature_type": "fbank",
   "testing": [
     1
   ],
   "training": [
     2
   ],
   "validation": [
     3
   ]
 }' 'http://127.0.0.1:8080/v0.1/corpus'
```

Gives this:

```
{
  "feature_type": "fbank",
  "filesystem_path": "f2976bee-900f-11e8-9a5f-d8cb8acb264b",
  "id": 1,
  "label_type": "phonemes",
  "max_samples": null,
  "name": "Test Corpus",
  "preprocessed": false,
  "testing": [
    1
  ],
  "training": [
    1
  ],
  "validation": [
    1
  ]
}
```

The test, train, validation are the primary keys of the DataSet DB models, NOT the utterance primary keys.

**Comment by shuttle1987** [comment]
Thinking about this more I'm not sure if the best thing is to return ID's of utterances or training sets. The mismatch between the API advertising it is returning utterance keys but not is still definitely a bug though.

**Comment by aapeliv** [comment]
The [API spec](https://github.com/persephone-tools/persephone-web-API/blob/1ead1a71cb0aa43486a2b90d61696d008ce116c8/persephone_api/api_spec.yaml#L459-L500) claims that these should be [objects](https://github.com/persephone-tools/persephone-web-API/blob/1ead1a71cb0aa43486a2b90d61696d008ce116c8/persephone_api/api_spec.yaml#L618-L630) with a bunch of other info; not just IDs.

**Comment by shuttle1987** [comment]
I believe this is fixed as of #163
